src/tests/topp/ConvertTSVToTraML_2_output.TraML

```diff
@@ -113,7 +111,6 @@
         <cvParam cvRef="MS" accession="MS:1000827" name="isolation window target m/z" value="501" unitCvRef="MS" unitAccession="MS:1000040" unitName="m/z"/>
       </Precursor>
       <Product>
-        <cvParam cvRef="MS" accession="MS:1000041" name="charge state" value="2"/>
```

Comment:
We previously didn't propagate this information from TSV to TraML at all. The product charge state was set to the precursor charge state when doing the conversion as far as I understood it.  My suggestion would be to be more explicit about this and potentially to even require product charge in the TSV. This might require some changes in spectrast2tsv.py though. ConvertTSVToTraML is already capable to parse SpectraST annotation and set the CV terms accordingly.